webplugin/js/app/kommunicate-ui.js

```diff
@@ -588,7 +594,38 @@ handleAttachmentIconVisibility : function(enableAttachment, msg, groupReloaded)
             startConversationButton.classList.add('force-n-vis');
             hasMultipleConversations ? backButton.classList.remove('force-n-vis') : backButton.classList.add('force-n-vis')
         }
-    }
+    },
+        handleWaitingQueueMessage: function () {
+            let group = CURRENT_GROUP_DATA;
+            let groupId = group && group.tabId;
+            let waitingStatus = group && group.conversationStatus == Kommunicate.conversationHelper.status.WAITING;
+            window.Applozic.ALApiService.ajax({
+                type: 'GET',
+                url: MCK_BASE_URL + '/rest/ws/group/waiting/list',
+                global: false,
+                contentType: 'application/json',
+                success: function (res) {
+                    if(res.status === "success"){
+                    WAITING_QUEUE = res.response;
+                    let waitingQueueNumber= document.getElementById('waiting-queue-number');
```

Comment:
use var

Reply:
@Kusum-Dhoundiyal Do these changes in "Development" and "plugin-6.4" branch.